csm/conf/setup.yaml

```diff
@@ -54,13 +53,13 @@ csm:
 
     backup:
         files:
-            - /etc/csm/csm.conf 
-            - /etc/csm/database.yaml 
+            - /etc/csm/csm.conf
+            - /etc/csm/database.yaml
             - /etc/csm/components.yaml
             - /etc/csm/cluster.conf
             - /etc/ssl/stx/stx.pem
 
     refresh_config:
```

Comment:
If refresh_config is not used anywhere we should remove it. Confirm from Nitin.

Reply:
Yes Working on the Same in Different Ticket and Will be Removing it in seperate Ticket.
These Code Changes appeared since my Main branch was one commit behind.